scripts/globals/summon.lua

```diff
@@ -10,112 +10,278 @@ function getSummoningSkillOverCap(avatar)
+        fSTR = (dSTR + 6) / 4
+    elseif dSTR >= 1 then
+        fSTR = (dSTR + 7) / 4
+    elseif dSTR >= -2 then
+        fSTR = (dSTR + 8) / 4
+    elseif dSTR >= -7 then
+        fSTR = (dSTR + 9) / 4
+    elseif dSTR >= -15 then
+        fSTR = (dSTR + 10) / 4
+    elseif dSTR >= -21 then
+        fSTR = (dif + 12) / 4
+    else
+        fSTR = (dif + 13) / 4
+    end
+
+    local min = math.floor(weaponDmg/9)
+    return math.max(-min, fSTR)
+end
+
+function avatarHitDmg(weaponDmg, fSTR, pDif)
+    -- https://www.bg-wiki.com/bg/Physical_Damage
+    -- Physical Damage = Base Damage * pDIF
+    -- where Base Damange is defined as Weapon Damage + fSTR
+    return (weaponDmg + fSTR) * pDif
+end
+
 function AvatarPhysicalMove(avatar, target, skill, numberofhits, accmod, dmgmod, dmgmodsubsequent, tpeffect, mtp100,
     mtp200, mtp300)
     local returninfo = {}
 
-    local acc = avatar:getACC() + utils.clamp(getSummoningSkillOverCap(avatar), 0, 200)
+    -- I have never read a limit on accuracy bonus from summoning skill which can currently go far past 200 over cap
+    -- current retail is over +250 skill so I am removing the cap, my SMN is at 695 total skill
+    local acc = avatar:getACC() + getSummoningSkillOverCap(avatar)
     local eva = target:getEVA()
-    local dmg = avatar:getWeaponDmg()
-    local minFstr, maxFstr = avatarFSTR(avatar:getStat(tpz.mod.STR), target:getStat(tpz.mod.VIT))
-    local ratio = avatar:getStat(tpz.mod.ATT) / target:getStat(tpz.mod.DEF)
-    local firstacc = acc + 100
-
-    -- formula should be 75 + floor( (Accuracy - Evasion)÷2 ) - 2×(dLVL)
-    --  level correction does not apply .
-    -- https://web.archive.org/web/20200905213200if_/https://kegsay.livejournal.com/tag/smn!
-    local hitdiff = 0
-    local hitrate = 75
-    local firsthit = 0
-
-    hitdiff = hitrate + math.floor(acc - eva)
-    firsthit = hitrate + math.floor(firstacc - eva)
-
-    hitrate = hitdiff / 100
-    firsthit = firsthit / 100
-    hitrate = utils.clamp(hitrate, 0.2, 0.95)
-    firsthit = utils.clamp(firsthit, 0.2, 0.95)
-
-    -- add on native crit hit rate (guesstimated, it actually follows an exponential curve)
-    local critrate = (avatar:getStat(tpz.mod.DEX) - target:getStat(tpz.mod.AGI)) * 0.005 -- assumes +0.5% crit rate per 1 dDEX
-    critrate = critrate + avatar:getMod(tpz.mod.CRITHITRATE) / 100
-    critrate = utils.clamp(critrate, 0.05, 0.2)
-
-    -- Applying pDIF
-    if ratio <= 1 then
-        maxRatio = 1
-        minRatio = 1 / 3
-    elseif ratio < 1.6 then
-        maxRatio = (2 * ratio + 1) / 3
-        minRatio = (7 * ratio - 4) / 9
-    elseif ratio <= 1.8 then
-        maxRatio = 1.8
-        minRatio = 1
-    elseif ratio < 3.6 then
-        maxRatio = 2.4 * ratio - 2.52
-        minRatio = 5 * ratio / 3 - 2
-    else
-        maxRatio = 4.2
-        minRatio = 4
+
+    local zoneId = avatar:getZone():getID()
+
+    local shouldApplyLevelCorrection = (zoneId < 256) and not (zoneId == 183)
```

Comment:
lol sure I'll add it, I left out the link because I'd been linking to pDIF so often. I thought I was safe!
I would agree with a zone flag, the first place I looked was the DB to see if there was any flag per expansion and or anything else I could use. Luckily they're all clumped together except for Legion A